src/usecases/deleteVisitByCallId.test.js

```diff
@@ -0,0 +1,41 @@
+import deleteVisitByCallId from "./deleteVisitByCallId";
+
+describe("deleteVisitByCallId", () => {
+  it("returns a json object containing the result", async () => {
+    const container = {
+      async getDb() {
+        return {
+          any: jest.fn().mockReturnValue([
+            {
+              sucess: true,
+            },
+          ]),
+        };
+      },
+    };
+
+    const { result, error } = await deleteVisitByCallId(container)(
+      "cb238rfv23cuv3"
+    );
+
+    expect(error).toBeNull();
+    expect(result).toEqual({
+      success: true,
+    });
+  });
+
+  it("returns an error object on db exception", async () => {
+    const container = {
+      async getDb() {
+        return {
+          any: jest.fn(() => {
+            throw new Error("DB Error!");
+          }),
+        };
+      },
+    };
+
+    const { error } = await deleteVisitByCallId(container)("cb238rfv23cuv3");
+    expect(error).toBeDefined();
```

Comment:
In the code I **_think_** we return the error message (guessing `error.toString()` does that, not sure), so maybe we can do an expectation that we return that instead of checking if it's been defined.

```suggestion
    expect(error).toEqual("DB Error!");
```

Reply:
Agree :)